requirements.txt

```diff
@@ -0,0 +1,2 @@
+BeautifulSoup==3.2.1
+wsgiref==0.1.2
```

Comment:
I think _urllib_ and _urllib2_ are core modules and are available as part of python so they are not really needed there.